First scripts

kjrcda opened the pull request:
made first database erase scripts for mac and windows

kjrcda committed:
First scripts

made first database erase scripts for mac and windows

paymog commented on webapp/Scripts/README.txt:
```diff
@@ -0,0 +1,4 @@
+ERASE
+To erase the database you need to run the Erase_DB script
+	Windows - Erase_DB.bat
+	Mac	- Earase_DB.sh
```
for mac we'd need to run `sh Erase_DB.sh`

paymog commented:
can you rename `Scripts` to `scripts` (lowercase) and do the same with the files?

kjrcda committed:
specifics

renamed files and folder
added updated mac command to run script
Not sure if names will change., i see no update in my commit here

kjrcda committed:
Rename Erase_DB.bat to erase_db.bat

kjrcda committed:
Rename Erase_DB.sh to erase_db.sh

kjrcda commented:
Made the file name changes but had to do it on GitHub site. Not sure how to do it with the folder name though...

paymog commented:
http://www.patrick-wied.at/blog/rename-files-and-folders-with-git

kjrcda committed:
Thanks @paymogh for the think. This is nasty to do on my machine casue its not case sensitive. It would switch and then not recognizethe change sometimes lol. got it now though.
